<launch>

    <include file="$(find sensors)/launch/asmc.launch" />

    <include file="$(find sensors)/launch/usv_simulation.launch" />

    <node pkg="sensors" type="waypoints.py" name="waypoints" />

    <node name="rl_guidance" pkg="sensors" type="rl_guidance.py" />

    <!--Launch rosbag to record topics -->
    <!--node type="record" pkg="rosbag" name="recorder" args="record -a -o /home/alex/Documents/rasp_ws/src/sensors/bags/rl_g/arc2_2/w_speed/straight_1" /-->

</launch>
